package.json

```diff
@@ -13,7 +13,7 @@
     "postinstall": "lerna run postinstall && lerna run prepublish",
     "pretty-check": "prettier --check \"src/**/*.{md,mdx,html,json,yml,js}\"",
     "publish": "scripts/publish.sh",
-    "start:ec": "start-storybook -p 9001 -c ./src/ec/.storybook",
+    "start:ec": "start-storybook -p 8082 -c ./src/ec/.storybook",
```

Comment:
is there a reason why you changed this port? This may affect other people working on the project too.

Reply:
The reason is I'm working on Cloud9 and there is no port 9001 on it.
but this file does not be pushed.
I fix it

Reply:
If you have to change the port, that's not really an issue, but you just have to syncronize with other people working on the project to ensure that it won't cause trouble.
I don't recommand to put this file in gitignore anyways.